caster/lib/settings.py

```diff
@@ -100,6 +100,7 @@ def _find_natspeak():
         "flashdevelop": True,
         "foxitreader": True,
         "gitbash": True,
+        "gitter": True,
```

Comment:
This isn't really a problem for you. This is just sort of a reminder for myself. I need to make it one of my top priorities to change the configuration system so that core files aren't changed whenever a minor configuration item is added.